stackdriver/alert_policies.tf

```diff
@@ -0,0 +1,283 @@
+resource "google_monitoring_alert_policy" "high_throttled_write_ops" {
+  project      = "${var.gce_project}"
```

Comment:
DO you think we can use launch this with Terraform 0.12?

Reply:
I would rather keep everything on 0.11 until we have a full plan for migrating to 0.12